server/controllers/events/events_controller.go

```diff
@@ -306,9 +307,18 @@ func (e *VCSEventsController) HandleGithubCommentEvent(event *github.IssueCommen
 		}
 	}
 
+	body := event.GetComment().GetBody()
+
+	if strings.HasPrefix(body, "atlantis ") {
```

Comment:
I think we can customize the execution name now so we should avoid hard coding this